bundler/lib/bundler/dependency.rb

```diff
@@ -158,5 +159,13 @@ def specific?
     rescue NoMethodError
       requirement != ">= 0"
     end
+
+    private
+
+    def default_force_ruby_platform
```

Comment:
I'd like to suggest this:
```suggestion
    # Note that TruffleRuby (and possibly other Ruby implementations) might override this method for more flexibility.
    # This default implementation is provided for compatibility with older TruffleRuby releases.
    def default_force_ruby_platform
```

Reply:
As I said above, let's discuss this when/if you decide to override the method in the future. It seems clear to me now what the method is doing.

Reply:
OK. It wasn't clear to me initially on this PR, so I think such a comment would be helpful for anyone else reading this code.